### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by yerrism1977** [code-ready/crc#2755]
### General information

  * OS: Windows 10
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it (Yes/No)? Yes
  * Running CRC on: Laptop

## CRC version
```bash
# Put `crc version` output here
CodeReady Containers version: 1.31.2+19538dab
OpenShift version: 4.8.4 (not embedded in executable)
  
## CRC status
```bash
# Put `crc status --log-level debug` output here
CRC VM:          Stopped
OpenShift:       Stopped (v4.8.4)
Disk Usage:      0B of 0B (Inside the CRC VM)
Cache Usage:     14.38GB
Cache Directory: C:\Users\Y9F99V754\.crc\cache

## CRC config
```bash

Modifies crc configuration properties.
Properties:

* autostart-tray                    Automatically start the tray (true/false, default: true)
* bundle                            Bundle path (string, default 'C:\Program Files\CodeReady Containers\crc_hyperv_4.8.4.crcbundle')
* consent-telemetry                 Consent to collection of anonymous usage data (yes/no)
* cpus                              Number of CPU cores (must be greater than or equal to '4')
* disable-update-check              Disable update check (true/false, default: false)
* disk-size                         Total size in GiB of the disk (must be greater than or equal to '31')
* enable-cluster-monitoring         Enable cluster monitoring Operator (true/false, default: false)
* enable-experimental-features      Enable experimental features (true/false, default: false)
* host-network-access               Allow TCP/IP connections from the CodeReady Containers VM to services running on the host (true/false, default: false)
* http-proxy                        HTTP proxy URL (string, like 'http://my-proxy.com:8443')
* https-proxy                       HTTPS proxy URL (string, like 'https://my-proxy.com:8443')
* kubeadmin-password                User defined kubeadmin password
* memory                            Memory size in MiB (must be greater than or equal to '9216')
* nameserver                        IPv4 address of nameserver (string, like '1.1.1.1 or 8.8.8.8')
* network-mode                      Network mode (user or system)
* no-proxy                          Hosts, ipv4 addresses or CIDR which do not use a proxy (string, comma-separated list such as '127.0.0.1,192.168.100.1/24')
* proxy-ca-file                     Path to an HTTPS proxy certificate authority (CA)
* pull-secret-file                  Path of image pull secret (download from https://cloud.redhat.com/openshift/create/local)
* skip-check-administrator-user     Skip preflight check (true/false, default: false)
* skip-check-bundle-extracted       Skip preflight check (true/false, default: false)
* skip-check-crc-users-group-exists Skip preflight check (true/false, default: false)
* skip-check-hyperv-installed       Skip preflight check (true/false, default: false)
* skip-check-hyperv-service-running Skip preflight check (true/false, default: false)
* skip-check-hyperv-switch          Skip preflight check (true/false, default: false)
* skip-check-user-in-hyperv-group   Skip preflight check (true/false, default: false)
* skip-check-vsock                  Skip preflight check (true/false, default: false)
* skip-check-windows-edition        Skip preflight check (true/false, default: false)
* skip-check-windows-version        Skip preflight check (true/false, default: false)

Usage:
  crc config SUBCOMMAND [flags]
  crc config [command]

Available Commands:
  get         Get a crc configuration property
  set         Set a crc configuration property
  unset       Unset a crc configuration property
  view        Display all assigned crc configuration properties

Flags:
  -h, --help   help for config

Global Flags:
      --log-level string   log level (e.g. "debug | info | warn | error") (default "info")

Use "crc config [command] --help" for more information about a command.

# Put `crc config view` output here
- consent-telemetry                     : no
- cpus                                  : 8
- disk-size                             : 100
- memory                                : 16216

## Host Operating System
```bash
# Put the output of `systeminfo` in case of Windows
Host Name:                 DESKTOP-131EF8A
OS Name:                   Microsoft Windows 10 Enterprise
OS Version:                10.0.19043 N/A Build 19043
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          IBMUSER
Registered Organization:   IBM
Product ID:                00330-80000-00000-AA309
Original Install Date:     30/06/2021, 23:45:38
System Boot Time:          16/09/2021, 10:03:40
System Manufacturer:       LENOVO
System Model:              20MAS17205
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 158 Stepping 10 GenuineIntel ~2592 Mhz
BIOS Version:              LENOVO N2CET59W (1.42 ), 27/04/2021
Windows Directory:         C:\windows
System Directory:          C:\windows\system32
Boot Device:               \Device\HarddiskVolume1
System Locale:             en-us;English (United States)
Input Locale:              en-gb;English (United Kingdom)
Time Zone:                 (UTC+01:00) Amsterdam, Berlin, Bern, Rome, Stockholm, Vienna
Total Physical Memory:     32,478 MB
Available Physical Memory: 19,184 MB
Virtual Memory: Max Size:  37,342 MB
Virtual Memory: Available: 10,283 MB
Virtual Memory: In Use:    27,059 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\DESKTOP-131EF8A
Hotfix(s):                 9 Hotfix(s) Installed.
                           [01]: KB4601050
                           [02]: KB4562830
                           [03]: KB4570334
                           [04]: KB4577266
                           [05]: KB4580325
                           [06]: KB4586864
                           [07]: KB5000736
                           [08]: KB5005565
                           [09]: KB5005699
Network Card(s):           7 NIC(s) Installed.
                           [01]: Intel(R) Wireless-AC 9560 160MHz
                                 Connection Name: Wi-Fi
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.1.254
                                 IP address(es)
                                 [01]: 192.168.1.136
                                 [02]: fe80::385d:33c8:cb05:6761
                                 [03]: 2001:b07:644b:2e8f:7096:6d28:9fe4:fe0d
                                 [04]: 2001:b07:644b:2e8f:385d:33c8:cb05:6761
                           [02]: Intel(R) Ethernet Connection (7) I219-LM
                                 Connection Name: Ethernet 2
                                 Status:          Media disconnected
                           [03]: Bluetooth Device (Personal Area Network)
                                 Connection Name: Bluetooth Network Connection
                                 Status:          Media disconnected
                           [04]: Cisco AnyConnect Secure Mobility Client Virtual Miniport Adapter for Windows x64
                                 Connection Name: Ethernet 3
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 9.171.30.202
                                 [02]: fe80::1d9c:d7a:9dcf:526e
                           [05]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (Default Switch)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 172.26.112.1
                                 [02]: fe80::c171:e81e:5c8c:c769
                           [06]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (WSL)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.48.1
                                 [02]: fe80::7475:210:7552:651d
                           [07]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (crc)
                                 Status:          Media disconnected
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.
```

### Steps to reproduce

  1. When installing too many containers the Windows hosts file grows on just one line and it breaks the host resolution

### Expected
hosts file to be broken into multiple lines
127.0.0.1 ibm-licensing-service-instance-ibm-common-services.apps-crc.testing
127.0.0.1 api.crc.testing
127.0.0.1 canary-openshift-ingress-canary.apps-crc.testing
127.0.0.1 console-openshift-console.apps-crc.testing
127.0.0.1 default-route-openshift-image-registry.apps-crc.testing
127.0.0.1 downloads-openshift-console.apps-crc.testing
127.0.0.1 oauth-openshift.apps-crc.testing
127.0.0.1 ftm-db2-dev-ftm-demo-ftm-demo.apps-crc.testing
127.0.0.1 ftm-mq-qm-ftm-demo-ftm-demo.apps-crc.testing
127.0.0.1 ftm-mq-web-ftm-demo-ftm-demo.apps-crc.testing
127.0.0.1 ftm-base-core-flows-app-ftm-demo-ftm-demo.apps-crc.testing
### Actual
127.0.0.1 ibm-licensing-service-instance-ibm-common-services.apps-crc.testing api.crc.testing canary-openshift-ingress-canary.apps-crc.testing console-openshift-console.apps-crc.testing default-route-openshift-image-registry.apps-crc.testing downloads-openshift-console.apps-crc.testing oauth-openshift.apps-crc.testing ftm-db2-dev-ftm-demo-ftm-demo.apps-crc.testing ftm-mq-qm-ftm-demo-ftm-demo.apps-crc.testing ftm-mq-web-ftm-demo-ftm-demo.apps-crc.testing ftm-base-core-flows-app-ftm-demo-ftm-demo.apps-crc.testing

**Comment by praveenkumar** [comment]
duplicate of https://github.com/code-ready/crc/issues/2710

/close

**Comment by gbraad** [comment]
Duplicate #2710

**Comment by gbraad** [comment]
Duplicate of #2710
